assets/tvs/multitv/includes/multitv.class.php

```diff
@@ -345,12 +345,15 @@ function renderTemplate($template, $placeholder)
     }
 
     // invoke modx renderFormElement and change the output (to multiTV demands)
-    function renderMultiTVFormElement($fieldType, $fieldName, $fieldElements, $fieldClass, $fieldDefault)
+    function renderMultiTVFormElement($fieldType, $fieldName, $fieldElements, $fieldClass, $fieldDefault, $fields=array())
```

Comment:
Thanks for your commit.

The additional parameter is not needed. You have $this->fields available inside and you should get the theme i.e. by $this->fields[$fieldName]['theme']. I don't know why you do that substr on $fieldName, it looks wrong, since $fieldClass contains the 'mtv_' prefix.